### PR TITLE
chore: Add support for Symfony 7

### PR DESCRIPTION
**Pull request by VincentLanglet**
Cf https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7431#issuecomment-1822364577

### CODE REVIEW

**Comment by keradus** [comment]
please also update CI file https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7431/files#diff-dd66bbfa032066c896a1083fd11952c1866f145ec2fba70ce25ec225bded54b0R59

**Comment by Wirone** [comment]
Looks like [`symfony/phpunit-bridge`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/6955481288/job/18924327914?pr=7453#step:3:564) is installed with 6.4 version, the other `symfony/*` components are in `7.0.0-RC1`.

File: composer.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "phpspec/prophecy-phpunit": "^2.0",
         "phpunit/phpunit": "^9.5",
         "symfony/phpunit-bridge": "^6.2.3",
```
Comment (Wirone) [inline composer.json:52]:
```suggestion
        "symfony/phpunit-bridge": "^6.2.3 || ^7.0",
```

Comment (VincentLanglet) [reply composer.json]: Fixed

File: .github/composite-actions/setup-php-with-composer-deps/action.yml [hunk 2]
```diff
@@ -56,7 +56,6 @@ runs:
       shell: bash
       run: |
         composer config minimum-stability beta
```
Comment (keradus) [inline .github/composite-actions/setup-php-with-composer-deps/action.yml:58]: RC ?

**Review by keradus — APPROVED** [review]

**Comment by Wirone** [comment]
Thank you @VincentLanglet 🍻

**Pull request merged**